src/components/EarnDashboard/components/StakingPositions/LpPositions.tsx

```diff
@@ -0,0 +1,197 @@
+import { Avatar, Button, Flex, Stack } from '@chakra-ui/react'
+import { Tag } from '@chakra-ui/tag'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { cosmosAssetId, cosmosChainId, fromAssetId, osmosisChainId } from '@shapeshiftoss/caip'
+import {
+  DefiAction,
+  DefiProviderMetadata,
+} from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import qs from 'qs'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory, useLocation } from 'react-router'
+import type { Column, Row } from 'react-table'
+import { Amount } from 'components/Amount/Amount'
+import { ReactTable } from 'components/ReactTable/ReactTable'
+import { RawText } from 'components/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import type { LpEarnOpportunityType, OpportunityId } from 'state/slices/opportunitiesSlice/types'
+import { getUnderlyingAssetIdsBalances } from 'state/slices/opportunitiesSlice/utils'
+import {
+  selectAggregatedEarnUserLpOpportunities,
+  selectAssets,
+  selectCryptoMarketData,
+  selectFirstAccountIdByChainId,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type ProviderPositionProps = {
+  ids: OpportunityId[]
+  assetId: AssetId
+}
+
+export type RowProps = Row<LpEarnOpportunityType>
+
+export const LpPositions: React.FC<ProviderPositionProps> = ({ ids, assetId }) => {
+  const translate = useTranslate()
+  const location = useLocation()
+  const history = useHistory()
+  const {
+    state: { isConnected, isDemoWallet },
+    dispatch,
+  } = useWallet()
+  const assets = useAppSelector(selectAssets)
+  const marketData = useAppSelector(selectCryptoMarketData)
+  const lpOpportunities = useAppSelector(selectAggregatedEarnUserLpOpportunities)
+  const filteredDown = lpOpportunities.filter(e => ids.includes(e.assetId as OpportunityId))
+
+  const cosmosAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, cosmosChainId),
+  )
+  const osmosisAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, osmosisChainId),
+  )
+
+  const handleClick = useCallback(
+    (opportunity: RowProps, action: DefiAction) => {
+      const {
+        original: {
+          type,
+          provider,
+          contractAddress,
+          chainId,
+          rewardAddress,
+          assetId,
+          highestBalanceAccountAddress,
+        },
+      } = opportunity
+      const { assetReference, assetNamespace } = fromAssetId(assetId)
+      const defaultAccountId = assetId === cosmosAssetId ? cosmosAccountId : osmosisAccountId
```

Comment:
We shouldn't pass this:
- `defaultAccountId` is a weird prop which was added to "fix" something something Cosmos, and should never have been there in the first place - `accountId` is valid though but not here, see below
- We're selecting aggregated opportunities here, so there shouldn't be any need to route with an `AccountId` either way